src/Core/Services/StateService.cs

```diff
@@ -843,6 +865,26 @@ public async Task SetThemeAsync(string value, string userId = null)
             await SetValueAsync(key, value, reconciledOptions);
         }
 
+        public async Task ApplyThemeGloballyAsync(string value)
+        {
+            // TODO remove this method (ApplyThemeGlobally) to restore per-account theme support
+            await CheckStateAsync();
+            if (_state?.Accounts != null)
+            {
+                var activeUserId = await GetActiveUserIdAsync();
+                foreach (var account in _state.Accounts)
+                {
+                    var uid = account.Value?.Profile?.UserId;
+                    if (uid == null || uid == activeUserId)
+                    {
+                        // active user theme already set, skip
+                        continue;
+                    }
+                    await SetThemeAsync(value, uid);
+                }
+            }
```

Comment:
In order to avoid nesting and some lines, I'd rewrite this as:
```c#
if (_state?.Accounts == null)
{
    return;
}

var activeUserId = await GetActiveUserIdAsync();
foreach (var account in _state.Accounts)
{
    var uid = account.Value?.Profile?.UserId;
    if (uid != null && uid != activeUserId)
    {
        // active user theme not already set, so set it
        await SetThemeAsync(value, uid);
    }
}
```